api/v1/services/RegistrationService.js

```diff
@@ -410,15 +407,15 @@ module.exports.updateAttendee = (attendee, attributes) => {
   // some attendee registration attributes are optional, but we need to
   // be sure that they are at least considered for removal during adjustment
   attributes = _.merge(attributes, {
-    'ecosystemInterests': [],
-    'projects': [],
-    'extras': [],
-    'collaborators': []
+    'longForm': [],
+    'collaborators': [],
+    'extraInfo': [],
+    'osContributors': []
   });
 
-  if (!_hasValidAttendeeAssignment(attributes.projects, attributes.ecosystemInterests)) {
-    const message = 'One project or ecosystem interest must be provided';
-    const source = ['projects', 'ecosystemInterests'];
+  if (!_hasValidAttendeeAssignment()) {
```

Comment:
This is also never going to be true

Reply:
Now checks osContributors 6a2e2c84e513e9b0f05760bc0318d111b439fc34